web/docs/understand/formats/xsv.md

```diff
@@ -0,0 +1,69 @@
+# xsv
+
+Reads and writes lines with separated values.
+
+## Synopsis
+
+```
+csv
+ssv
+tsv
```

Comment:
What about `xsv` itself (and its options)?

Reply:
Good point. I looked at the implementation again to figure this out.

What strikes me here as odd is that you need to specify _all 3 or none_ of the arguments. That doesn't match my user workflow. I typically want to selectively specify and override the defaults.

That said, I think I'm going to document just the status quo. Once the new configuration framework is there and we ditch the hand-rolled option parsing, we can revisit the `xsv` UX and turn the positional arguments in to flags/options.

Reply:
I think the rationale was that the `csv`, `ssv` and `tsv` aliases should cover most use cases. If you have a use case which doesn't fit into these, the chance is high that our defaults are no sufficient anyways. Also, there are some mutual dependencies here, for example space separation means that you must print something for nulls, but printing nothing should be the default for comma separation (etc). But yes, we should revisit all argument parsing once we have the proper framework for this.